src/inmanta/protocol/methods.py

```diff
@@ -876,7 +876,7 @@ def upload_code_batched(tid: uuid.UUID, id: int, resources: dict):
     :param tid: The id of the environment to which the code belongs.
     :param id: The version number of the configuration model.
     :param resources: A dictionary where each key is a string representing a resource type.
-                  For each resource type, the value is a dictionary. This nested dictionary's keys are file names,
+                  For each resource type, the value is a dictionary. This nested dictionary's keys are file hashes,
```

Comment:
I think they're technically both, but perhaps this is an improvement.

Reply:
To me `names` implies a clear text file name

Reply:
Well, it is. Iirc this is the name we use to store this on the file system. But perhaps I don't remember correctly.

Reply:
Yes I think these keys are hashes: we use the hash to call into upload_code_batched:  https://github.com/inmanta/inmanta-core/blob/a6b57a5ba24d9bccbee9884482de0510a5ab153e/src/inmanta/export.py#L107-L112